trieste/acquisition/rule.py

```diff
@@ -94,27 +88,39 @@ def __init__(
         self,
         builder: AcquisitionFunctionBuilder | None = None,
         optimizer: AcquisitionOptimizer[SP_contra] | None = None,
+        num_query_points: int = 1,
     ):
         """
         :param builder: The acquisition function builder to use. Defaults to
-            :class:`~trieste.acquisition.ExpectedImprovement` with tag :data:`OBJECTIVE`.
+                :class:`~trieste.acquisition.ExpectedImprovement` with tag :data:`OBJECTIVE`.
         :param optimizer: The optimizer with which to optimize the acquisition function built by
-            ``builder``. This should *maximize* the acquisition function, and must be compatible
-            with the global search space. Defaults to
-            :func:`~trieste.acquisition.optimizer.optimize`.
+                ``builder``. This should *maximize* the acquisition function, and must be compatible
+                with the global search space. Defaults to
+                :func:`~trieste.acquisition.optimizer.optimize`.
+        :param num_query_points: The number of points to acquire.
         """
+
+        if not num_query_points > 0:
+            raise ValueError(
+                f"Number of query points must be greater than 0, got {num_query_points}"
+            )
+
         if builder is None:
-            builder = ExpectedImprovement().using(OBJECTIVE)
+            if num_query_points == 1:
+                builder = ExpectedImprovement().using(OBJECTIVE)
+            else:
+                raise ValueError("Need to specify a batch acquisition function")
```

Comment:
I'm for dropping the default. Simpler. What do you think?

Reply:
Yep will do, although it means I will have to fiddle the notebooks

Reply:
well it's not very important but I find it convenient. Should we make that part of another PR, where we also try to get rid of .using(OBJECTIVE) when there is a single model?

Reply:
Yep. Lets leave for now

Reply:
it seems odd to me to say "the builder's optional" then error when you don't specify it

Reply:
but then perhaps it's no more odd than accepting a batch size only to error when the builder doesn't work with it